### DISCUSSION TITLE
How to change the HxInputdate format

### DISCUSSION THREAD

**Discussion by Guiprirod** [havit/Havit.Blazor#815] [Q&A]
Hello, I am new using Havit.

I am from Spain and I would like to know that is possible to change the format of a HxInputdate
.
 I would like to change the format to dd/MM/YYYY and the months in Spanish too. Is it possible?

Thank you for your time :)

![image](https://github.com/havit/Havit.Blazor/assets/105490638/5efccfcf-dbc5-4ab8-bd95-5fcfc4dd9764)
![image](https://github.com/havit/Havit.Blazor/assets/105490638/3840631d-6c6b-402e-9117-caf47341e8ef)

![image](https://github.com/havit/Havit.Blazor/assets/105490638/bd6abb48-8917-4088-826c-0c8a056e66fe)

**Comment by hakenr** [comment]
You have to set `CultureInfo.CurrentCulture` and `CultureInfo.CurrentUICulture` to your locale.
See https://learn.microsoft.com/en-us/aspnet/core/blazor/globalization-localization?view=aspnetcore-8.0
